.github/workflows/sync-backport-changelog.yml

```diff
@@ -18,7 +27,13 @@ jobs:
               run: |
```

Comment:
We can skip checking out the repo to check for changes if it's triggered by the label

Reply:
We have to check out the repo because of the `awk` in the script below. We can skip the `git diff` though. I've done that.